doc/user/ripd.rst

```diff
@@ -149,7 +149,7 @@ RIP Configuration
 
    The default is to be passive on all interfaces.
 
-.. clicmd:: ip split-horizon
+.. clicmd:: ip split-horizon [poisoned-reverse]
```

Comment:
We need to define what `poisoned-reverse` means too?

Reply:
Done, please help check it. Thanks.